kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/LessonResourceSelection/constants.js

```diff
@@ -0,0 +1,11 @@
+export const ResourceSelectionView = {
```

Comment:
I havent yet decided the files location, for now I have left all these new files within the LessonSummaryPage folder, but some of them will probably need to be relocated.